pgx_async.opam

```diff
@@ -11,7 +11,7 @@ install: [jbuilder install]
 remove: [jbuilder uninstall]
 
 depends: [
-  "async" {>= "v0.9.0"}
+  "async" {>= "v0.10.0"}
```

Comment:
Can we actually not compile against async v0.9 if we land this?

Reply:
No, since there are breaking API changes.